src/mixins/Projector.ts

```diff
@@ -215,14 +218,22 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(base: T)
 		}
 
 		public setChildren(children: DNode[]): void {
+			this._projectorChildren = [ ...children ];
 			super.__setChildren__(children);
 		}
 
 		public setProperties(properties: P & { [index: string]: any }): void {
+			this._projectorProperties = assign({}, properties);
```

Comment:
Hmmm... I couldn't find anything on TypeScript issues, so I opened this: https://github.com/Microsoft/TypeScript/issues/15792